Userland/Libraries/LibC/stdlib.cpp

```diff
@@ -844,9 +844,15 @@ char* mktemp(char* pattern)
 
 // https://pubs.opengroup.org/onlinepubs/9699919799/functions/mkstemp.html
 int mkstemp(char* pattern)
+{
+    return mkstemps(pattern, 0); // mkstemp is mkstemps with suffix_length 0.
```

Comment:
I'd argue that the comment here is a bit superfluous (since that's _exactly_ what the code next to it expresses). Also, if anything, the comment should go above the code that it comments, not behind it.

Reply:
Alright, I moved this information to the commit message.